tests/integration/steps/common_steps.py

```diff
@@ -3,17 +3,21 @@
 from assertpy import assert_that
 import json
 from utilities.config_reader import read_config
+import allure
 
 
 @when("I request data for {params} from {resource_name}")
 def send_get_with_params(context, params, resource_name):
+    allure.dynamic.description("Workspace: " + context.workspace)
```

Comment:
Does this dynamic label need to be included in everystep? Can it be at a higher level - for example as a background step that is included in every feature file?